### PR TITLE
Update Boards content/v2

### PR DESCRIPTION
**Pull request by justinegeffen**
Boards content review, polish, update.

@chenilim, this is a polish of the content. Next up is the addition of content around the workspace (this will be under Accessing your workspace). I'm also going through the closed PRs to pick up any additional content needed.

### CODE REVIEW

**Review by cwarnermm — APPROVED** [review]
Great updates!

**Comment by justinegeffen** [comment]
@chenilim, reminder on this for review when you have a chance. :)

File: source/boards/working-with-boards.rst [hunk 1]
```diff
@@ -1,18 +1,55 @@
 Working with Boards
 ===================
 
-Adding new Boards
------------------
+Using Boards begins with selecting the type of board you want to use. The containers within a board, called cards, are used for tasks. Views are a way to group cards based on their property (e.g., priority, status, etc).
```
Comment (chenilim) [inline source/boards/working-with-boards.rst:4]: Recommend not introducing a new concept of "containers" here. Maybe: "A board contains cards, which typically track tasks or topics, and views, which define how to display the cards, or a subset of them. Views can display cards in a board, table, or gallery layout, optionally filtered and grouped by a property (e.g., priority, status, etc)."

File: source/boards/sharing-boards.rst [hunk 2]
```diff
@@ -1,11 +1,11 @@
 Sharing Boards
 ==============
 
-1. Hover over the right-hand side of the board menu.
-2. Click on the options menu (…) and select “Share board”. 
-3. Turn the switch on which says “Publish to web and share this board with anyone”. 
-4. Copy the given link.
+1. Open a board in any view.
```
[the remark this answers is not in the document]
Comment (justinegeffen) [reply source/boards/sharing-boards.rst:4]: Thanks, @chenilim! Do you have mocks of what the System Console will look like with this setting? We don't currently have any System Console settings so a section for permissions and defaults etc would be a good idea. 

Let's table that for v3 of the docs which will be written for 6.0, if that works for you?

File: source/boards/templates.rst [hunk 3]
```diff
@@ -1,31 +1,43 @@
...
-2. Click on the options menu (… at the top right)
-3. Select “New template from card”
+1. Open the card you want to use as a template.
+2. Select the options menu **...** in the top-right corner of the card.
+3. Select **New template from card**.
+4. Edit the card as needed, including a helpful name.
+5. Close the card using the **X** in the top left corner.
+6. Select the drop-down arrow next to **New**, then select the card you just created.
 
+The card is applicable only to the board in which it's created and isn't available in other boards in your workspace.
```
Comment (justinegeffen) [inline source/boards/templates.rst:43]:
```suggestion
The card template is applicable only to the board in which it's created and isn't available in other boards in your workspace.
```

File: source/boards/working-with-boards.rst (hunk 1 above)
Comment (justinegeffen) [inline source/boards/working-with-boards.rst:4]:
```suggestion
Using Boards begins with selecting the type of board you want to use. A board contains cards, which typically track tasks or topics, and views, which define how to display the cards, or a subset of them. Views can display cards in a board, table, or gallery layout, optionally filtered and grouped by a property (e.g., priority, status, etc).
```

**Pull request merged**